WORKSPACE

```diff
@@ -46,6 +48,28 @@ load("@bazel_skylib//lib:unittest.bzl", "register_unittest_toolchains")
 
 register_unittest_toolchains()
 
+############################################
+# Stardoc
+load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
+
+stardoc_repositories()
+
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
+
+rules_jvm_external_deps()
+
+load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
+
+rules_jvm_external_setup()
```

Comment:
What change brought in all the stardoc+jvm requirements?

Reply:
Latest stardoc now depends on rules_jvm_external

Reply:
and stardoc doesn't have a `stardoc_deps()` or something like that?